plugins/goodreads.koplugin/goodreadsbook.lua

```diff
@@ -190,17 +190,19 @@ function GoodreadsBook:genBookInfoGroup()
         align = "top",
         HorizontalSpan:new{ width =  split_span_width }
     }
-    --thumbnail
+    -- thumbnail
     local http = require("socket.http")
     local body = http.request(self.dates.image)
     local image = false
     if body then image = Pic.openJPGDocumentFromMem(body) end
     if image then
         table.insert(book_info_group, ImageWidget:new{
-            image = image.image_bb,
+            image_disposable = false,
+            image = image.image_bb:copy(),
```

Comment:
oops, good catch, sorry i forgot to delete that line. It was added without the copy. Then I noticed it won't work, so I switched to the copy approach.